hack/test-end-to-end-scenario.sh

```diff
@@ -18,6 +18,7 @@ function wait_for_app() {
 	echo "[INFO] Waiting for app in namespace $1"
 	echo "[INFO] Waiting for database pod to start"
 	wait_for_command "oc get -n $1 pods -l name=database | grep -i Running" $((60*TIME_SEC))
+	oc deploy database --logs -n $1
```

Comment:
There might be a very small window for a race condition here: if the deployment succeeds, the deployer will get deleted, which *could* happen before this call to view its logs. Maybe add a comment here so that if this ever produces flakes, we'll be able to quickly diagnose and refactor it?